server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -96,17 +111,28 @@ def mock_results_push(self, ctrl, tb, token):
 
     def test_process_tb(self, monkeypatch):
         """verify processing of tar balls without any failure"""
+        tb_path = fixture_path / "tarball"
+
+        def mock_remove(*args):
+            pass
```

Comment:
Wo-hoo!  Go you!  😀

However, once again, I think it would be best if the [signature](https://docs.python.org/3/library/os.html#os.remove) matched exactly:
```
        def mock_remove(path, *, dir_fd=None):
```
And, it might be good to verify that the input path is what we expect, based on what `mock_glob()` returned.